x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/index.tsx

```diff
@@ -4,92 +4,67 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { useState, useEffect, useCallback } from 'react';
-import type { SavedObjectAttributes } from '@kbn/securitysolution-io-ts-alerting-types';
+import { useEffect, useMemo } from 'react';
 import type { TiDataSources } from './use_ti_data_sources';
-import type { LinkPanelListItem } from '../../components/link_panel';
 import { useKibana } from '../../../common/lib/kibana';
+import { getDashboardsByTagIds } from '../../../common/containers/dashboards/api';
+import { getTagsByName } from '../../../common/containers/tags/api';
+import { useFetch, REQUEST_NAMES } from '../../../common/hooks/use_fetch';
+import { SecurityPageName } from '../../../../common/constants';
+import { useGetSecuritySolutionUrl } from '../../../common/components/link_to';
 
-const TAG_REQUEST_BODY_SEARCH = 'threat intel';
-export const TAG_REQUEST_BODY = {
-  type: 'tag',
-  search: TAG_REQUEST_BODY_SEARCH,
-  searchFields: ['name'],
+const CTI_TAG_NAME = 'threat intel';
+
+const useCtiInstalledDashboards = () => {
+  const { http } = useKibana().services;
+
+  const {
+    fetch,
+    data: dashboards,
+    isLoading,
+  } = useFetch(REQUEST_NAMES.CTI_TAGS, async (tagName: string) => {
+    const ctiTags = await getTagsByName(http, tagName);
+    return getDashboardsByTagIds(
+      http,
+      ctiTags.map((tag) => tag.id)
+    );
+  });
+
+  useEffect(() => {
+    fetch(CTI_TAG_NAME);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Does `fetch` instance changes? If it doesn't we can add it to the dependency array and get rid of `eslint-disable-next-line`.